packages/dlp/src/index.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+var gapic = {
+ v2beta1: require('./v2beta1'),
+};
+
+const VERSION = require('../package.json').version;
+
+/**
+ * Create an dlpServiceClient with additional helpers for common
+ * tasks.
+ *
+ * @param {Object=} opts - The optional parameters.
```

Comment:
Rename `opts` to `options`, and you can just paste the JSDocs from here: https://github.com/GoogleCloudPlatform/google-cloud-node/blob/d152416b48928ab13a17f710425266ed1b82dce0/packages/video-intelligence/src/index.js#L52

It doesn't include custom types that the user won't be familiar with (`grpc.ClientCredentials=` and `gax.constructSettings`). An alternative solution would be to add links to what those are.